collector/lambdacomponents/go.mod

```diff
@@ -2,4 +2,13 @@ module github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents
 
 go 1.16
 
-require go.opentelemetry.io/collector v0.33.0
+require (
+  go.opentelemetry.io/collector v0.36.0
+  github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.36.0
```

Comment:
Just wondering where did this list come from?

Reply:
It's based on all the components here that were no longer in the core collector: https://github.com/open-telemetry/opentelemetry-lambda/blob/main/collector/lambdacomponents/default.go#L19-L30